src/test/java/BaseTest.java

```diff
@@ -1,18 +1,60 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.safari.SafariDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
 
+import java.sql.Driver;
+import java.time.Duration;
 
-public class BaseTest {
 
+public class BaseTest {
+    WebDriver driver;
+  
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
+    @BeforeMethod
+    @Parameters({"baseUrl"})
+    public void setUpBrowser(String baseUrl){
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+        driver.get("baseUrl");
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/223664023-7847a8b1-e236-4d6c-a691-451a25417b75.png)

________________________________________________________________________________________________________________________

Hint

```
  WebDriver driver;
    String url;
  
    @BeforeSuite
    static void setupClass() {
        WebDriverManager.chromedriver().setup();
    }
    @BeforeMethod
    @Parameters({"baseUrl"})
    public void setUpBrowser(String baseUrl){
        driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.manage().window().maximize();
        url=baseUrl;
        driver.get(url);
        
    }
```

Reply:
ok, i updated to this... not sure why it still failing?